package.json

```diff
@@ -10,7 +10,7 @@
     "lint": "eslint \"*.js\" \"./src/**/**/*.js*\"",
     "release": "npm run clean && npm run lint && webpack --config ./webpack.config.release.js",
     "serve": "npm run build && http-server ./build -o --proxy http://stage.pvdgeeks.org",
-    "test": "npm run clean && jest --config ./jest.config.json"
+    "test": "npm run clean && TZ=EST jest --config ./jest.config.json"
```

Comment:
Jest provides a means of setting globals via a `jest` object. 

https://jestjs.io/docs/en/configuration#globals-object

Reply:
No luck trying this. Went the RegEx approach instead.